src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -26,6 +26,7 @@
         :fields.sync="options.fields"
         max-height="max-height: 45vh;"
```

Comment:
Dont forget to remove the redundant max heights

Reply:
thank you